### DISCUSSION TITLE
My mail template send with same format to user

### DISCUSSION THREAD

**Discussion by salmanshafiq8630** [pimcore/pimcore#13904] [Q&A / Support] [answered]
Hi All,

and my mail template send with same format to mail

![image](https://user-images.githubusercontent.com/47385962/209542434-dd1de68c-9761-4c2a-ad79-e9737798dc36.png)

help me asap

**Comment by mattamon** [comment, marked as answer]
Hey @salmanshafiq8630 ,

please see my answer here: https://github.com/pimcore/pimcore/discussions/13759#discussioncomment-4501003
